src/Native/Unix/System.Security.Cryptography.Native.Apple/pal_ssl.c

```diff
@@ -20,14 +20,19 @@ SSLContextRef AppleCryptoNative_SslCreateContext(int32_t isServer)
 {
     if (isServer != 0 && isServer != 1)
         return NULL;
-
+#pragma clang diagnostic push
```

Comment:
Nit: please add back the whitespace that was there before, e.g. there should ideally be a blank line above this pragma

Reply:
will update with all similar

Reply:
i assume i should apply it for master and cherry pick to this PR ...

Reply:
for such a trivial thing, you need not wait for the same change to merge into master.

Reply:
yeah, just in case commits need to be identical, but if that is not needed, will update right here

Reply:
updated here because its easier to review